### DISCUSSION TITLE
Using different environment variable values per Server

### DISCUSSION THREAD

**Discussion by jonasbark** [basecamp/kamal#1185] [Q&A]
I went through the docs, issues, and discussions as I felt that what I had in mind was more common:

I've got two servers to which I want to deploy the same docker image but with different environment variables *values* (secrets). In this case, one server is for the US region, the other for EU and they need to use different values.

My current approach is using two destinations (eu, us), which works but is probably not ideal here.

I saw the `--skip-push` option for `kamal deploy` that probably uses the latest docker image (documentation is unclear here) and at least prevents the generation of a new docker image.

**Comment by tuladhar** [comment]
Here’s a suggestion that might help.


First, deploy to one server, tagging the Docker image (e.g: v1):
```
kamal deploy -d us --version 1
```

Next, deploy to the second server using the same Docker image with the `--skip-push` option:
```
kamal deploy -d eu --version 1 --skip-push
```
